deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -617,6 +617,12 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
                         self.messageInputBar.isHidden = true
                     }
                 }
+
+                let oldMuted = self.navigationItem.rightBarButtonItems?.contains(self.muteItem)
+                let newMuted = self.dcChat.isMuted
+                if oldMuted != newMuted {
+                    self.updateTitle()
```

Comment:
I would just call `updateTitle()` here. On a first glance the extra code around it seems to optimize at a place where there's not so much to gain.

Reply:
i was hesitant as things are a bit of a mess for other things, eg. the member list counter or name changes are updated somehow somewhere else ... and i wanted to avoid double updates.

but maybe the correct way is to remove these updates (so, probably mostly the call to updateTitle() on viewWillAppear()

Reply:
i added a commit